test/parallel/test-timers-clear-timeout-interval-equivalent.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const common = require('../common');
+
+// This test makes sure that timers created with setTimeout can be disarmed by
+// clearInterval and that timers created with setInterval can be disarmed by
+// clearTimeout.
+//
+// This behavior is documented in Node.js' documentation and the HTML Living
+// standard:
+//
+// * Refs: https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html#dom-setinterval
+// * Refs: https://nodejs.org/api/timers.html#timers_class_timeout
+
+// Disarm interval with clearTimeout.
+const interval = setInterval(common.mustNotCall(), common.platformTimeout(20));
```

Comment:
Nit: `common.platformTimeout(20)` can just become `1` now. We only use `common.platformTimeout` when the test is timing sensitive. Since in this test no timeouts will execute — unless it's broken — it doesn't need that extra overhead.